src/sync/incomingSyncUtils.js

```diff
@@ -130,6 +132,14 @@ export const sanityCheckIncomingRecord = (recordType, record) => {
       cannotBeBlank: ['item_ID', 'store_ID'],
       canBeBlank: [],
     },
+    Location: {
+      cannotBeBlank: [],
```

Comment:
@joshxg 
I am not a big fan of this sanity check, i know it's there to protect the integrity of incoming data but it will silently skip records (records which may be important), I had to add blank objects here because to pass the sanity check. Still though, good point i'll update. by the way, i'll fix this while i am at it
![Screen Shot 2019-04-09 at 2 47 59 PM](https://user-images.githubusercontent.com/26194949/55769908-8a38f300-5ad6-11e9-8fb0-8a39d876bf1b.png)